### DISCUSSION TITLE
Cilium Egress Gateway (via Floating IPs) on existing Cluster?

### DISCUSSION THREAD

**Discussion by zypriafl** [kube-hetzner/terraform-hcloud-kube-hetzner#664] [Q&A]
Hello,

I was wondering if "Cilium Egress Gateway (via Floating IPs)" from the "examples" can be added to an existing ClusteR?

**Comment by mysticaltech** [comment]
@zypriafl I believe the easiest way would be to create a new nodepool to add it, and then if scaling down your previous nodepool to 0 if you do not need that much computing power.
